docs/libcurl/opts/CURLOPT_SSL_EC_CURVES.3

```diff
@@ -0,0 +1,53 @@
+.\" **************************************************************************
+.\" *                                  _   _ ____  _
+.\" *  Project                     ___| | | |  _ \| |
+.\" *                             / __| | | | |_) | |
+.\" *                            | (__| |_| |  _ <| |___
+.\" *                             \___|\___/|_| \_\_____|
+.\" *
+.\" * Copyright (C) 1998 - 2017, Daniel Stenberg, <daniel@haxx.se>, et al.
```

Comment:
nitpicking: maybe better by replacing `2017` by `2020`?

Reply:
Seems correct, yes.

Note: we have a script ([scripts/copyright.pl](https://github.com/curl/curl/blob/master/scripts/copyright.pl)) that detects such discrepancies which we will run before the next release so copyright year range mistakes will be fixed.

Reply:
@bagder Thanks also for this information: I take it then that I don't need to update the PR but that your script will take care of this issue. Please let me know if I'm wrong with this understanding.

Reply:
If you're going for a gold star PR you fix it, since then nobody else has to, but if you don't there's this safety net that should kick in... :grinning:

Reply:
What a question -- who doesn't want to go for gold? :-)